pkg/util/sys/device.go

```diff
@@ -79,6 +87,8 @@ type LocalDisk struct {
 	WWNVendorExtension string `json:"wwnVendorExtension"`
 	// Empty checks whether the device is completely empty
 	Empty bool `json:"empty"`
+	// Information provided by Ceph Volume Inventory
+	CephVolumeData *CephVolumeDeviceData `json:"cephVolumeData,omitempty"`
```

Comment:
After reading #2606 again, I don't think that adding these properties to the LocalDisk type will answer the question for that issue. The intention was that we would only need to call c-v to get the disk properties instead of also calling lsblk or other tools to get the properties. @leseb @sebastian-philipp wdyt? Does the dashboard expect the raw c-v output?

Reply:
I think that the requirements are exposed in: https://github.com/rook/rook/issues/2606#issuecomment-510861208
AFAIK, the dashboard only needs this properties:
`"cephVolumeData":{"device path", "size" ,"rotates","available","model name":}}`

But also is true that if we need more information in the future we are going to expect that this information will come from c-v ... so maybe we will need to include all the c-v output in the configmap. (inside CephVolumeDeviceData )
This is not nice because we will be repeating information in the configmap. A possible solution will be generate a different configmaps for each node with the information coming  only from ceph-volume. In Rook Ceph we would use only c-v to get all the information, others backends can continue using the current configmaps with the current logic to extract the data.